spinalcordtoolbox/scripts/sct_maths.py

```diff
@@ -483,6 +547,21 @@ def main(argv: Sequence[str]):
     verbose = arguments.v
     set_loglevel(verbose=verbose)
 
+    # Check data types
+    current_type = 'number' if isinstance(arguments.i, float) else 'image'
```

Comment:
I think it doesn't support it currently, but I wanted to add support specifically for two use cases:
- negating an image with `sct_maths -i 0 -sub img.nii.gz`
- inverting an image with `sct_maths -i 1 -div img.nii.gz`

I'm willing to accept that it's too much added complexity, though. (And it could probably be addressed with new `-neg` and `-inv` operators.)